### DISCUSSION TITLE
Login failed

### DISCUSSION THREAD

**Discussion by Zinski** [MiSchroe/ioBroker.klf200#85] [Q&A] [answered]
**Describe the bug**  
Wenn ich versuche, mich in meiner neuen Installation des ioBroker (in Docker) mit dem Adapter zu verbinden, bekomme ich immer ein Login failed als Ergebnis im Log. 
Ich habe mittlerweile alle Konstellationen durch - denke ich zumindest - und komme nicht weiter.
Ich würde gerne einmal versuchen mit Postman die API des KLF anzusprechen, weiß jedoch nicht was genau ich dafür ausführen muss.

Leider ist die Dokumentation etwas unscharf hinsichtlich der Konfiguration:
Mal ist die Eingabe des Hosts mit einem http:// mal ohne,
Mal heißt es das WebLogon passwort (standard **velux123**) müsse dafür genutzt werden, mal der Schlüssel des WLANs.
Vielleicht - und das ist meine Hoffnung - liegt es an der Konfig.

Zur Umgebung kann ich sagen, dass der ioBroker im Docker Container läuft, er ist in einem MACVLAN und hat Internetzugriff. Ich kann die KLF aus dem Container heraus anpingen und mich auch per telnet auf 51200 verbinden. Das Alles klappt. Auch fahren die Fenster und Rollos, wenn ich die potentialfreien Kontakte am KLF benutze. Irgendwas an der Verbindung kommt nicht zurecht.

@MiSchroe könntest du mir da vllt. helfen?

**Versions:**  
 - Adapter version: 1.0.1
 - JS-Controller version: 3.3.22
 - Node version: 14.18.3
 - Operating system: Debian/RaspberryOS
 - KLF Firmware: 2.0.0.71

**Comment by zaphod2** [comment]
Feste LAN IP vergeben und diese dann im Adapter eintragen ohne http:// ... und das Wlan-Passwort von hinten auf der KLF200. So läuft es zumindest erstmal bei mir mit 6 min. abbrüchen, aber verbindet sich.

**Comment by Zinski** [comment]
danke für die info, aber es klappt nicht. ich bekomme nur die meldung login failed.
das gerät antwortet auf den ping, aber der adapter kommt nicht drauf.
ich habe jetzt alles mögliche versucht.

was ich als zusatzinfo noch geben kann: ich habe 3 KLF 200 hier und mit keiner klappt es. muss ich auf der KLF noch irgendwas konfigurieren, was ich vllt. überlesen habe?

**Comment by ThKls** [comment]
Hast Du wirklich das WLAN-Passwort genommen (nicht das velux123)?  Bei mit hat bei einer solchen Situation mit mehreren Fehlversuchen der Neustart des KLF geholfen. Hatte dann auch die Instanz gelöscht und neu eingerichtet. Ist in der KLF die LAN-Schnittstelle auf Dauerbetrieb?

**Comment by Zinski** [comment]
Leider ja... ich habe wirklich alle Kombinationen durch.
Was ich noch nicht gemacht habe, dass ich die KLF mal komplett resete und dann von vorne anfange... in meinem Unifi seh ich Velux höchstens mal online, aber ohne IP

**Comment by Zinski** [comment]
ich habe jetzt die KLF komplett zurückgesetzt, 2 geräte (ein rollo ein fenster) angelernt, die ip im adapter eingetragen, das WLAN pw der box eingetragen und wieder kommt login failed.

![image](https://user-images.githubusercontent.com/8176856/152643862-ec22ba57-3673-41eb-ab63-b72c851c9172.png)

@MiSchroe ich verzweifle langsam und weiß nicht wo der fehler liegt. kannst du mir hier irgendwie helfen?

**Comment by Zinski** [comment, marked as answer]
ich habe jetzt den iobroker mal auf einem eigenen pi4 aufgesetzt und das ganze versucht (um docker auschließen zu können... ich komme auch von da nicht auf das KLF... mein KLF blinkt weiß, kennt jemand die bedeutungen der LED?

**Reply by ThKls** [reply]
MiSchroe hat ein weiteres Repository: klf-200-api. Das bindet er auch ein. Schau mal da rein. Vielleicht kannst Du Dich da stückchenweise rantasten. Unter Certificates ist beschrieben, wie das Zertifikat der KLF-200 extrahiert wird. Damit könntest Du schauen, ob überhaupt der TLS-Tunnel aufgebaut wird. Hast Du mal das Logging auf Debug gesetzt und kannst das posten?

**Reply by MiSchroe** [reply]
Blinken sollte da eigentlich gar nichts. Bei mir ist die LED dauerhaft weiß.

**Reply by Zinski** [reply]
Hallo zusammen,
gestern hat mir ein Kumpel geholfen... er hatte einen Blick von Außen auf die Sache. Ich sagte ihm, dass mich das Geblinke irritiere, und dann guckte er im Manual nach und sagte nach etwas Zeit, hast du mal den Knopf nach dem Suchen gedrückt?
Hatte ich nicht, er drückte ihn, das Blinken wurde zu einem weißen Leichten und es ging.
